### DISCUSSION TITLE
Flashing or blinking in DeckGl

### DISCUSSION THREAD

**Discussion by bjvesely** [visgl/deck.gl#5612] [Q&A]
Is there a way to do flashing or blinking of objects in DeckGL, such as the have a object in the  polygon layer blink on and off.

**Comment by AdriSolid** [comment]
You can always play with properties like `opacity` or `visible` to achieve this. Also with `transitions`.

[Here](https://codepen.io/AdriSolid/pen/poRJeLe) there is an example setting an interval and `visible` property to `true` or `false`

**Reply by ibgreen** [reply]
If you are adventurous, you can subclass the layer and modify the fragment shader. You can just output another color when some condition/uniform is set. There are pretty good docs for how to do that. 

If it is just one object you could play with highlight color.
